### ISSUE TITLE
Force Order on Rollover doesn't work after multi-range update

### ISSUE THREAD

**Issue by sharkbit** [sharkbit/BadgeApp#79]

**Comment by sharkbit** [comment]
- Updated the way I was checking if events have peen published to the next year.
- Found an Issue when creating a new recurring event, when rollover is automatic, event master record is not placed in new year.

**Comment by sharkbit** [comment]
code -https://github.com/sharkbit/BadgeApp/commit/c894ed2efe5c9c7cfc41e12638fd3cea1a618e3b

**Comment by sharkbit** [comment]
I believe it's Good...  Needs Lots of testing...   using rollover and normal recurring events.

**Comment by tdasto** [comment]
I unpublished and republished a couple of club multi-range events. no issues.

Then, I intentionally created a conflict to see if the App could catch it and it didn't. Here is what I did;

1. I unpublished Action events with your function.
2. Created a one-time event, see ID=52373. 
3. Then, republished ID=52374.
4. No conflicts created in Conflicts tab.
5. Both events are listing on 7/11/2021.
6. Open both in separate tabs and check availability. Finds conflict in each. Still no events in Conflicts tab.

I would think that the App should give priority to 52374 and knock 52373 off the calendar and into the Conflicts tab.
